whipper/program/cdrdao.py

```diff
@@ -63,10 +167,7 @@ def DetectCdr(device):
     cmd = [CDRDAO, 'disk-info', '-v1', '--device', device]
     logger.debug("executing %r", cmd)
     p = Popen(cmd, stdout=PIPE, stderr=PIPE)
-    if 'CD-R medium          : n/a' in p.stdout.read():
-        return False
-    else:
-        return True
+    return 'CD-R medium          : n/a' not in p.stdout.read()
```

Comment:
Is this related to this PR? Seems like the underlying logic of this isn't changed, just how it's written.

Reply:
No. @JoeLametta did that. It behaves identically in my testing with a pressed CD but is simplified.

Reply:
Right. My point is that it's a change that doesn't belong in this PR, even if the fault is not yours by @JoeLametta's. It adds on to the code that reviewers have to look through, even though it isn't related to the topic of "Refactor cdrdao toc/table functions into Task and provide progress output". It should be its own PR. (Small PRs are okay too, in fact, I'd rather review a lot of small, self-contained PRs than a bunch of big ones containing random other fixes.)

@JoeLametta?

Reply:
Sorry, I've reverted this change.